src/components/Card/Card.story.js

```diff
@@ -15,7 +15,22 @@ storiesOf('Card', module)
   .addDecorator(withTests('Card'))
   .add(
     'Card',
-    withInfo()(() => <Card style={{ width: '500px', height: '300px' }} />)
+    withInfo()(() => (
+      <Fragment>
+        <Card
+          shadow={Card.SINGLE}
+          style={{ width: '500px', height: '150px', marginBottom: '15px' }}
+        />
+        <Card
+          shadow={Card.DOUBLE}
+          style={{ width: '500px', height: '150px', marginBottom: '15px' }}
+        />
+        <Card
+          shadow={Card.TRIPLE}
+          style={{ width: '500px', height: '150px', marginBottom: '15px' }}
+        />
+      </Fragment>
+    ))
```

Comment:
Missing the spacings story.

Reply:
Should we make one story for the default card, one for the various shadows and one for the spacings? This way each modifier gets a story?